package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hubot-cf-convenience",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
@clanzen I believe either the author of the fix/patch can do it or the person doing the publish can do it.  Just as long as they are in sync.  In this case I knew I wanted to republish so I went ahead and did it.